modules/ROOT/pages/dashboards/visualizations/barchart.adoc

```diff
@@ -0,0 +1,128 @@
+
+|Min Value |Number |Auto |If not set to "auto", this value is minimum value for the bar chart.
+
+|Max Value |Number |Auto |If not set to "auto", this value is the maximum value for the bar chart.
+
+|Group Mode |List |Stacked |This setting determines how different groups are visualized when grouping is enabled.
+If set to stacked, different groups of the same category are stacked on top of each other.
+If set to grouped, they are placed alongside each other.
+
+|Layout |List |Vertical |Whether to use a vertical or horizontal bar
+chart layout.
+
+|Color Scheme |List | |The color scheme to use for the category groups.
+Colors are assigned automatically (consecutively) to the different groups returned by the Cypher query.
+
+|Show values on Bars |on/off |off |If set, shows the category value inside the respective bar.
+
+|Skip label on width (px) |Mumber |0 |Skip the label if the bar width is lower than the provided value. Ignored if 0.
+
+|Skip label on height (px) |Number |0 |Skip the label if the bar height is lower than the provided value. Ignored if 0.
+
+|Custom label position |off/top/bottom |off | Allow user to place label out of the bar.
+This overrides any other label configuration.
+
+|Label Rotation (degrees) |Number |45 |The angle at which the bar labels
+are rotated.
+
+|Margin Left (px) |Number |50 |The margin in pixels on the left side of the visualization.
+
+|Margin Right (px) |Number |24 |The margin in pixels on the right side of the visualization.
+
+|Margin Top (px) |Number |24 |The margin in pixels on the top side of the visualization.
+
+|Margin Bottom (px) |Number |40 |The margin in pixels on the bottom side of the visualization.
+
+|Legend Width (px) |Number |128 |The width in pixels of each legend label on top of the visualization (if enabled).
+
+|Hide Selections |on/off |off |If set, hides the property selector (footer of the visualization).
+
+|Auto-run query |on/off |on |If set, automatically runs the query when the report is displayed.
+Otherwise, the query is displayed and must be executed manually.
+
+|Report Description |Markdown text | | If specified, adds a button the report header that opens a pop-up.
+The pop-up contains the rendered Markdown from this setting. 
+
+|Bar Width |Number |10 | Only active when "custom dimensions" is "on".
+The width of each bar.
+Increasing the bar width increases the width of the chart.
+This setting has the largest influence on the width of the chart.
+
+|Expand Height For Legend |on/off |off |Useful when the legend has many labels.
+If set, the chart height adjusts to the number of rows returned by the query to prevent legend labels from being cut off.
+
+|Inner Padding |Number |0 |If set, adds padding between grouped elements.
+
+|Legend Position |Vertical/horizontal |Vertical |Dictates whether the lagend is displayed vertically on the right hand side of the chart or horizontally on the bottom of the chart.
+|===
+////
+
+
+== Rule-based styling
```

Comment:
We can remove rule-based styling sections for each chart, this will come later in the year.

Reply:
Same goes for report actions btw